cluster/addons/calico-policy-controller/README.md

```diff
@@ -3,13 +3,6 @@
 
 Calico is an implementation of the Kubernetes network policy API.  The provided manifest installs a DaemonSet which runs Calico on each node in the cluster.
 
-### Templating
```

Comment:
I'm not sure I would conflate the two things here.  IP Masquerading I would argue is not really enforcing network policy, it was only there in Calico because since kubenet isn't being used in this case (because CNI is) there was nothing to do that masquerading.  Since masquerading has been decoupled from kubenet there really is not reason to conflate the two.